src/mmpars.c

```diff
@@ -4453,101 +4455,101 @@ vstring rewrapComment(const char *comment1) {
+      //   commentTemplate[pos] = ASCII_4;
+      // } else if (comment[pos + 1] == '`'
+      //     && strchr(OPENING_PUNCTUATION, comment[pos - 1]) != NULL) {
+      //   // Don't break space between punctuation and math start '`'
+      //   commentTemplate[pos] = ASCII_4;
+      // } else if (comment[pos - 1] == '`'
+      //     && strchr(CLOSING_PUNCTUATION, comment[pos + 1]) != NULL) {
+      //   // Don't break space between punctuation and math end '`'
+      //   commentTemplate[pos] = ASCII_4;
+
       } else if (comment[pos - 3] == ' ' && comment[pos - 2] == 'p'
           && comment[pos - 1] == '.') {
-        /* Don't break " p. nnn" */
+        // Don't break " p. nnn"
         commentTemplate[pos] = ASCII_4;
       }
     }
   }
-  commentTemplate[length - 3] = ASCII_4; /* Last space in comment */
+  commentTemplate[length - 3] = ASCII_4; // Last space in comment
 
   for (pos = 3; pos < length - 2; pos++) {
-    /* Transfer the non-breaking spaces from the template to the comment */
+    // Transfer the non-breaking spaces from the template to the comment
     if (commentTemplate[pos] == ASCII_4) comment[pos] = ASCII_4;
   }
 
   free_vstring(commentTemplate);
   return comment;
-} /* rewrapComment */
-
-/* This is a general-purpose function to parse a math token string,
-   typically input by the user at the keyboard.  The statemNum is
-   needed to provide a context to determine which symbols are active.
-   Lack of whitespace is tolerated according to standard rules.
-   g_mathTokens must be set to the proper value. */
-/* The code in this section is complex because it uses the fast parsing
-   method borrowed from parseStatements().  On the other hand, it must set
-   up some initial tables by scanning the entire g_MathToken array; this may
-   slow it down in some applications. */
-/* Warning:  g_mathTokens must be the correct value (some procedures might
-   artificially adjust g_mathTokens to add dummy tokens [schemes] to the
-   g_MathToken array) */
-/* The user text may include existing or new dummy variables of the
-   form "?nnn". */
+} // rewrapComment
+
+// This is a general-purpose function to parse a math token string,
+// typically input by the user at the keyboard.  The statemNum is
+// needed to provide a context to determine which symbols are active.
+// Lack of whitespace is tolerated according to standard rules.
+// g_mathTokens must be set to the proper value.
+// The code in this section is complex because it uses the fast parsing
+// method borrowed from parseStatements().  On the other hand, it must set
+// up some initial tables by scanning the entire g_MathToken array; this may
+// slow it down in some applications.
+// Warning:  g_mathTokens must be the correct value (some procedures might
+// artificially adjust g_mathTokens to add dummy tokens [schemes] to the
+// g_MathToken array).
+// The user text may include existing or new dummy variables of the
+// form "?nnn".
```

Comment:
```suggestion
// This is a general-purpose function to parse a math token string,
// typically input by the user at the keyboard.  The statemNum is
// needed to provide a context to determine which symbols are active.
// Lack of whitespace is tolerated according to standard rules.
// g_mathTokens must be set to the proper value.
//
// The code in this section is complex because it uses the fast parsing
// method borrowed from parseStatements().  On the other hand, it must set
// up some initial tables by scanning the entire g_MathToken array; this may
// slow it down in some applications.
//
// Warning:  g_mathTokens must be the correct value (some procedures might
// artificially adjust g_mathTokens to add dummy tokens [schemes] to the
// g_MathToken array).
//
// The user text may include existing or new dummy variables of the
// form "?nnn".
```